entries/b/bokio.se.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Bokio Sverige": {
+    "domain": "bokio.se",
+    "url": "https://www.bokio.se",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```